internal/providers/pke/pkeworkflow/select_volume_sizes_activity.go

```diff
@@ -0,0 +1,105 @@
+// Copyright © 2020 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkeworkflow
+
+import (
+	"context"
+
+	"emperror.dev/errors"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+const SelectVolumeSizesActivityName = "pke-get-ami-image-details-activity"
+const MinimalVolumeSize = 50 // TODO make this a config
+
+type SelectVolumeSizesActivity struct {
+	awsClientFactory *AWSClientFactory
+}
+
+type SelectVolumeSizesActivityInput struct {
+	AWSActivityInput
+	NodePools []NodePool
+}
+
+type AmiImageDetails struct {
```

Comment:
**Recommended**: you **SHOULD** consider renaming this type to AMIImageDetails because the [standard Golang naming convention](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) is to use consistent casing for identifiers.

I know we neglected this convention throughout the repository, but unless we decide otherwise, I would try converging towards a more standard looking code-base.

Reply:
Fixed in f0cea1608aba0595d4c986d69a710f437765911c.